pkg/config/config.go

```diff
@@ -271,7 +271,7 @@ func loadDotEnv() {
 	var err error
 	dotEnv, err = godotenv.Read(file)
 	if err != nil {
-		log.Warn(err)
+		log.Infof("No environment loaded from %s file: %s", file, err.Error())
```

Comment:
I feel like `open .env: no such file or directory` is not needed. How about `No environment loaded from .env file: No .env found`?

Reply:
Updated to: `No environment loaded from .env file: Not found`